demo/scrollable-page.html

```diff
@@ -31,7 +31,7 @@
     }
 
     .scrollmargin {
-        height: 500px;
+        height: 200px;
```

Comment:
```suggestion
        height: 1800px;
```

Reply:
NO it should be this 
```suggestion
        height: 900px;
```